recipes/fribidi/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+from conans import tools, ConanFile, Meson
+import glob
+import os
+
+
+class FriBiDiCOnan(ConanFile):
+    name = "fribidi"
+    description = "The Free Implementation of the Unicode Bidirectional Algorithm"
+    topics = ("conan", "fribidi", "unicode", "bidirectional", "text")
+    license = "LGPL-2.1"
+    homepage = "https://github.com/fribidi/fribidi"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_deprecated": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_deprecated": True,
+    }
+    exports_sources = "patches/**"
+
+    _meson = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def build_requirements(self):
+        if not tools.which("meson"):
+            self.build_requires("meson/0.53.0")
```

Comment:
```suggestion
        self.build_requires("meson/0.53.0")
```

Conan team discussed some time about about it. The problem is when some user open an issue and we can't reproduce the same behavior, because we don't same tools installed. cmake and compilers are exception, but for others we prefer explicit using from Conan packages.

Reply:
I remember something like that.
Must have forgotten that in the heat of cooking recipes. :smile: